src/components/literacy-material/literacy-material.resolver.ts

```diff
@@ -2,79 +2,73 @@ import { Args, Mutation, Query, Resolver } from '@nestjs/graphql';
 import { AnonSession, ID, IdArg, LoggedInSession, Session } from '../../common';
 import { Loader, LoaderOf } from '../../core';
 import {
-  CreateLiteracyMaterialInput,
-  CreateLiteracyMaterialOutput,
-  LiteracyMaterial,
-  LiteracyMaterialListInput,
-  LiteracyMaterialListOutput,
-  UpdateLiteracyMaterialInput,
-  UpdateLiteracyMaterialOutput,
-} from './dto';
-import { LiteracyMaterialLoader } from './literacy-material.loader';
-import { LiteracyMaterialService } from './literacy-material.service';
+  CreateEthnoArtInput,
+  CreateEthnoArtOutput,
+  EthnoArt,
+  EthnoArtListInput,
+  EthnoArtListOutput,
+  EthnoArtLoader,
+  EthnoArtService,
+  UpdateEthnoArtInput,
+  UpdateEthnoArtOutput,
+} from '../ethno-art';
+import { LiteracyMaterial } from './dto';
 
+/** @deprecated Use EthnoArtResolver instead */
 @Resolver(LiteracyMaterial)
 export class LiteracyMaterialResolver {
-  constructor(
-    private readonly literacyMaterialService: LiteracyMaterialService
-  ) {}
+  constructor(private readonly ethnoArtService: EthnoArtService) {}
 
-  @Query(() => LiteracyMaterial, {
+  @Query(() => EthnoArt, {
```

Comment:
I was trying to avoid breaking changes here. I think we should be able to return EthnoArt objects as "Songs" since they have the same shape. So these resolvers should be able to get by without any schema changes